package.json

```diff
@@ -102,8 +102,8 @@
     "react-focus-on": "^3.5.0",
     "react-input-autosize": "^2.2.2",
     "react-is": "~16.3.0",
-    "react-virtualized-auto-sizer": "^1.0.2",
-    "react-window": "^1.8.5",
+    "react-virtualized-auto-sizer": "1.0.7",
```

Comment:
According to the `npm` the new version was released relatively recently (https://www.npmjs.com/package/react-virtualized-auto-sizer?activeTab=versions), but nevertheless I cannot upgrade to the latest version because I get this error on `yarn start`

```
node_modules/react-virtualized-auto-sizer/dist/react-virtualized-auto-sizer.module.js 201:24
Module parse failed: Unexpected token (201:24)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
| 
| class $6e2bb92d9db3b0c8$export$9d94f4ee1d930ff extends (0, $hgUW1$Component) {
>     static defaultProps = {
|         onResize: ()=>{},
|         disableHeight: false,
 @ /Users/andreymyssak/Gomasko/Projects/opensearch/oui/src/components/selectable/selectable_list/selectable_list.tsx 73:0-53 350:49-58
 @ /Users/andreymyssak/Gomasko/Projects/opensearch/oui/src/components/selectable/selectable_list/index.ts
 @ /Users/andreymyssak/Gomasko/Projects/opensearch/oui/src/components/selectable/selectable.tsx
 @ /Users/andreymyssak/Gomasko/Projects/opensearch/oui/src/components/selectable/index.ts
 @ /Users/andreymyssak/Gomasko/Projects/opensearch/oui/src/components/index.js
 @ ./routes.js
 @ ./index.js
```

Reply:
Interesting. Maybe a newer Webpack loader would fix the issue. Might want to open a follow-up. What about `1.0.8`, same issue?

Reply:
Yes, the same

Reply:
Yeah, usually indicative of an issue with our Webpack/babel configuration. This is likely the best approach until we can do https://github.com/opensearch-project/oui/issues/584. But maybe it's worth making a follow-up issue for?